app/api/vector.py

```
@@ -0,0 +1,181 @@
+# app/api/vector.py - 최소화 버전 (정말 필요한 것만)
+from fastapi import APIRouter, HTTPException, Depends, BackgroundTasks
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy import select, func
+from typing import Dict, Any
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)